### PR TITLE
docker: reload container-feeder after starting docker

### PR DESCRIPTION
**Pull request by mssola**
Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

**Comment by flavio** [comment]
LGTM. One really minor note, maybe we should have a dedicated state for container-feeder to make it easier to be tracked (and removed) later on.

File: salt/docker/init.sls [hunk 1]
```diff
@@ -49,4 +49,3 @@ docker:
       - pkg: docker
       - file: /etc/sysconfig/docker
       - /etc/systemd/system/docker.service.d/proxy.conf
-
```
Comment (flavio) [inline salt/docker/init.sls]: This is an accidental change, please revert it

File: salt/top.sls [hunk 2]
```diff
@@ -13,6 +13,7 @@ base:
     - etcd-proxy
     - flannel
     - docker
+    - container-feeder
```
Comment (flavio) [inline salt/top.sls]: I think `container-feeder` is needed only on the minions (BTW we should really change the name to worker, but not right now...). Am I wrong?

Comment (inercia) [inline salt/top.sls]: We need `container-feeder` in all the machines where `docker` is installed, so I think it is fine as it is.

**Pull request merged**